charts/dependency-track/values.yaml

```diff
@@ -116,10 +129,23 @@ apiserver:
     # If not set and create is true, a name is generated using the fullname template
     name: apiserver-serviceaccount
   # See https://docs.dependencytrack.org/getting-started/configuration/ for backend configuration options.
-  readinessProbe:
-    initialDelaySeconds: 60
   livenessProbe:
+    enabled: true
+    path: "/"
```

Comment:
I think this path can be: https://github.com/DependencyTrack/dependency-track/issues/1001#issuecomment-814032659

Reply:
I'll make it so.